src/directives/ml-facets.directive.js

```diff
@@ -61,6 +61,14 @@
     $scope.truncateLength = parseInt(attrs.truncate) || 20;
     $scope.shouldShowMore = !!attrs.showMore;
     $scope.shouldNegate = !!attrs.negate && !!attrs.activeFacets;
+
+    $scope.$watch('facets', function(newVal, oldVal) {
+      _.each(newVal, function(facet) {
+        if ( facet.type === 'bucketed' ) {
+          facet.displayingAll = true;
```

Comment:
Sorry, was pointing to wrong line. I meant to say, you could also toggle this flag when you receive the data in the service, where metadata and such is rewritten..